install/host_preparation.adoc

```diff
@@ -20,14 +20,33 @@ ifdef::openshift-enterprise[]
 [[software-prerequisites]]
 == Operating system requirements
 
-Master and node hosts must use a base installation of Red Hat Enterprise Linux
-(RHEL) 7.4 or 7.5 with the latest packages from the Extras
-channel or RHEL Atomic Host 7.4.2 or later. See the following documentation for
-the respective installation instructions, if required:
+The operating system requirements for master and node hosts are different
+depending on your server architecture.
+
+* For servers that use x86_64 architecture, use a base installation of Red Hat
+Enterprise Linux (RHEL) 7.4 or 7.5 with the latest packages from the Extras
```

Comment:
We also support and tested rhel 7.6 and I believe x86 also supports this. Maybe that is another PR?

Reply:
For x86, the issue was reported on https://github.com/openshift/openshift-docs/issues/13012. I'll get our QE to confirm.